src/modules/content-tree/content.tree.module.js

```diff
@@ -43,6 +44,8 @@ export default class ContentTreeModule extends Component {
         loadSubtree(this.props.restInfo, this.subtree, (loadedSubtree) => {
             this.setInitialItemsState(loadedSubtree[0]);
         });
+
+        document.body.addEventListener('ctRefresh', this.refreshContentTree, false);
```

Comment:
I've chosen `ctRefresh` based on names such as: `pbIframeLoaded`, or` fbFormBuilderUnloaded`, but I don't think it is perfect. What do you think @dew326 ?

Reply:
I think we should prefix events with `ez-` like we have with `ez-notify` or `ez-bookmark-change` so for me the name should be `ez-content-tree-refresh`